inc/functions/files.php

```diff
@@ -1091,7 +1091,7 @@ function rocket_fetch_and_cache_busting( $src, $cache_busting_paths, $abspath_sr
 		 *
 		 * @param string The Document Root path.
 		*/
-		$document_root = apply_filters( 'rocket_min_documentRoot', ABSPATH );
+		$document_root = apply_filters( 'rocket_min_documentRoot', wp_normalize_path( dirname( $_SERVER['SCRIPT_FILENAME'] ) ) );
```

Comment:
While `ABSPATH` has a trailing slash, `dirname()` will remove it from `$_SERVER['SCRIPT_FILENAME']`. Is that an issue?

Reply:
The default value used in the Minify_CSS_Rewriter class is `$_SERVER['DOCUMENT_ROOT']` which doesn't have a trailing slash either, so it's fine to not have it either here